### PR TITLE
redone

### PR DESCRIPTION
**Pull request by LenaHenningsen**
> Pull-requests should be ready to merge, that is free of conflicts, and they should pass our testsuite. If you require a specific reviewer you can request a review on the right.

### Please indicate issues related to this PR (if any)
See # .
Close # .

### Checklist
-   The title line explains what these changes do
-   All my changes are marked in the `last_modfied` column
-   The PR has no merge conflicts
-   Travis checks are passing

### CODE REVIEW

File: data/PrimarySource.csv [hunk 1]
```diff
@@ -1165,3 +1165,5 @@ PS00705,秋水,,zh,P0797,G77,,,,,,,,,,,2019-02-24,WH,2019-02-25,WH
 PS00706,Zuo Zhuan,The First Year of Monarch Mingong,en,P0798,G56,,,,,,,,,This work was possibly wrote by him. The Chinese title is translated by project member.,,2019-02-24,WH,2019-02-24,WH
 PS00706,Zuo zhuan,Mingong yuannian,zh-Latn-pinyin,P0798,G56,,,,,,,,,This work was possibly wrote by him.,,2019-02-24,WH,2019-02-24,WH
 PS00706,左传,闵公元年,zh,P0798,G56,,,,,,,,,This work was possibly wrote by him.,,2019-02-24,WH,2019-02-24,WH
```
Comment (duncdrum) [inline data/PrimarySource.csv]:
```suggestion
PS00706,左传,闵公元年,zh,P0798,G56,,,,,,,,,This work was possibly written by him.,,2019-02-24,WH,2019-02-24,WH
```

File: data/PrimarySource.csv [hunk 2]
```diff
@@ -1165,3 +1165,5 @@ PS00705,秋水,,zh,P0797,G77,,,,,,,,,,,2019-02-24,WH,2019-02-25,WH
 PS00706,Zuo Zhuan,The First Year of Monarch Mingong,en,P0798,G56,,,,,,,,,This work was possibly wrote by him. The Chinese title is translated by project member.,,2019-02-24,WH,2019-02-24,WH
```
Comment (duncdrum) [inline data/PrimarySource.csv]:
```suggestion
PS00706,Zuo Zhuan,The First Year of Monarch Mingong,en,P0798,G56,,,,,,,,,This work was possibly written by him. The Chinese title is translated by project member.,,2019-02-24,WH,2019-02-24,WH
```

**Comment by LenaHenningsen** [comment]
Just did taht as you wrote…

Will now try to get travis working again.

 

 

Von: Duncan Paterson <notifications@github.com>
Antworten an: readchina/ReadingData <reply@reply.github.com>
Datum: Freitag, 8. März 2019 um 11:36
An: readchina/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Author <author@noreply.github.com>
Betreff: Re: [readchina/ReadingData] redone (#362)

 

@duncdrum requested changes on this pull request.

much better you need to get rid of the whitespace in the date lines 2019-03-08 instead of 2019-03-08 
and add the fact that you created the last entry in Primary_source

In data/PrimarySource.csv:
> @@ -1165,3 +1165,5 @@ PS00705,秋水,,zh,P0797,G77,,,,,,,,,,,2019-02-24,WH,2019-02-25,WH
 
 PS00706,Zuo Zhuan,The First Year of Monarch Mingong,en,P0798,G56,,,,,,,,,This work was possibly wrote by him. The Chinese title is translated by project member.,,2019-02-24,WH,2019-02-24,WH
 
 PS00706,Zuo zhuan,Mingong yuannian,zh-Latn-pinyin,P0798,G56,,,,,,,,,This work was possibly wrote by him.,,2019-02-24,WH,2019-02-24,WH
 
 PS00706,左传,闵公元年,zh,P0798,G56,,,,,,,,,This work was possibly wrote by him.,,2019-02-24,WH,2019-02-24,WH
 
⬇️ Suggested change 
- PS00706,左传,闵公元年,zh,P0798,G56,,,,,,,,,This work was possibly wrote by him.,,2019-02-24,WH,2019-02-24,WH
 
+PS00706,左传,闵公元年,zh,P0798,G56,,,,,,,,,This work was possibly written by him.,,2019-02-24,WH,2019-02-24,WH
 
In data/PrimarySource.csv:
> @@ -1165,3 +1165,5 @@ PS00705,秋水,,zh,P0797,G77,,,,,,,,,,,2019-02-24,WH,2019-02-25,WH
 
 PS00706,Zuo Zhuan,The First Year of Monarch Mingong,en,P0798,G56,,,,,,,,,This work was possibly wrote by him. The Chinese title is translated by project member.,,2019-02-24,WH,2019-02-24,WH
 
⬇️ Suggested change 
- PS00706,Zuo Zhuan,The First Year of Monarch Mingong,en,P0798,G56,,,,,,,,,This work was possibly wrote by him. The Chinese title is translated by project member.,,2019-02-24,WH,2019-02-24,WH
 
+PS00706,Zuo Zhuan,The First Year of Monarch Mingong,en,P0798,G56,,,,,,,,,This work was possibly written by him. The Chinese title is translated by project member.,,2019-02-24,WH,2019-02-24,WH

**Comment by LenaHenningsen** [comment]
I think the test went ok.

**Comment by LenaHenningsen** [comment]
Do I merge then or do you do that now?

 

Von: Duncan Paterson <notifications@github.com>
Antworten an: readchina/ReadingData <reply@reply.github.com>
Datum: Freitag, 8. März 2019 um 11:49
An: readchina/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Author <author@noreply.github.com>
Betreff: Re: [readchina/ReadingData] redone (#362)

 

@duncdrum approved this pull request.

**Comment by duncdrum** [comment]
once travis is green  I will

**Comment by LenaHenningsen** [comment]
Thanks a lot!

 

Von: Duncan Paterson <notifications@github.com>
Antworten an: readchina/ReadingData <reply@reply.github.com>
Datum: Freitag, 8. März 2019 um 11:51
An: readchina/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Author <author@noreply.github.com>
Betreff: Re: [readchina/ReadingData] redone (#362)

 

Merged #362 into master.

**Pull request merged**